x-pack/legacy/plugins/infra/public/components/logging/log_text_stream/scrollable_log_text_stream_view.tsx

```diff
@@ -92,7 +96,7 @@ export class ScrollableLogTextStreamView extends React.PureComponent<
         targetId: getStreamItemId(getStreamItemBeforeTimeKey(nextProps.items, nextProps.target!)),
         items: nextItems,
       };
-    } else if (!nextProps.target || !hasItems) {
```

Comment:
`target` (aka `position`, aka `cursor`, aka `timekey`) is no longer mandatory. If we only have a date range we show entries from the bottom.